package.json

```diff
@@ -1,94 +1,92 @@
 {
-	"name": "transactly",
-	"version": "2.0.4",
-	"description": "Transactional Key Value Store based on CosmosDB.",
-	"author": "Utily Contributors",
-	"license": "MIT",
-	"repository": "https://github.com/payfunc/transactly",
-	"bugs": {
-		"url": "https://github.com/payfunc/transactly/issues"
-	},
-	"homepage": "https://payfunc.com/open-source/transactly",
-	"private": false,
-	"main": "dist/index.js",
-	"typings": "dist/index.d.ts",
-	"git": {
-		"tagName": "v${version}"
-	},
-	"github": {
-		"release": true
-	},
-	"jest": {
-		"transform": {
-			"^.+\\.(j|t)sx?$": "ts-jest"
-		},
-		"transformIgnorePatterns": [
-			"node_modules/(?!(cryptly|authly|gracely|isoly|servly|cloud-router|cloud-http)/.*)"
-		],
-		"globals": {
-			"ts-jest": {
-				"tsconfig": "tsconfig.test.json",
-				"setTimeout": 2000
-			}
-		},
-		"testEnvironment": "node",
-		"testRegex": "((\\.|/)(test|spec))(\\.|\\/.+)(jsx?|tsx?)$",
-		"testPathIgnorePatterns": [
-			"node_modules/",
-			"dist/"
-		],
-		"collectCoverageFrom": [
-			"**/*.{ts,tsx,js,jsx}",
-			"!**/node_modules/**",
-			"!**/dist/**"
-		],
-		"preset": "ts-jest",
-		"testMatch": null
-	},
-	"scripts": {
-		"build": "tsc -p .",
-		"dev": "NODE_ENV=development npm run build",
-		"lint": "eslint '**/*.{ts,tsx}'",
-		"fix": "eslint '**/*.{ts,tsx}' --fix",
-		"clean": "rimraf dist node_modules coverage",
-		"test": "jest --maxWorkers=2",
-		"transpile": "tsc --project tsconfig.test.json",
-		"deploy": "wrangler publish",
-		"serve": "wrangler dev --env development",
-		"start": "npm run dev && KV_FILE_ROOT='../.kv/' cloudflare-worker-local  ./dist/worker.development.js localhost:3000 7150 ./variables.toml ",
-		"start.production": "NODE_ENV=production npm run build && KV_FILE_ROOT='../.kv/' cloudflare-worker-local ./dist/worker.production.js localhost:3000 7150 ./variables.toml "
-	},
-	"engines": {
-		"npm": ">=7.0.0",
-		"node": ">=16.0.0"
-	},
-	"devDependencies": {
-		"@cloudflare/workers-types": "2.2.2",
-		"@cloudflare/wrangler": "^1.19.12",
-		"@types/jest": "^28",
-		"@types/service-worker-mock": "^2.0.1",
-		"@typescript-eslint/eslint-plugin": "^5.23.0",
-		"@typescript-eslint/parser": "^5.23.0",
-		"babel-core": "^6.26.3",
-		"babel-jest": "^28",
-		"cloud-http": "^0.0.26",
-		"cloud-router": "^0.0.13",
-		"cloudflare-worker-local": "^1.13.0",
-		"dotenv": "^8.2.0",
-		"eslint": "^8.15.0",
-		"eslint-plugin-prettierx": "github:utily/eslint-plugin-prettierx#utily-20220323",
-		"eslint-plugin-simple-import-sort": "^7.0.0",
-		"isomorphic-fetch": "^3.0.0",
-		"jest": "^28",
-		"prettierx": "github:utily/prettierx#utily-20220323",
-		"rimraf": "^3.0.2",
-		"service-worker-mock": "^2.0.5",
-		"ts-jest": "^28",
-		"typescript": "^4.6.4"
-	},
-	"dependencies": {
-		"@cfworker/cosmos": "^1.6.11",
-		"cryptly": "3.0.1",
-		"isoly": "2.0.12"
-	}
+  "name": "transactly",
```

Comment:
Why change indentation?

Reply:
I dont know if they previous indentation was wrong but if it was you should not fix indentation and and update dependencies in the same commit/pr because then we will not see which dependencies you updated.

Reply:
Ok, that sentence was not completely understandable. 

But don't change the indentation, since it makes it difficult to see what was changed.

Reply:
This was auto changed, so I have to figure out how to not have vscode do that first. I'll take a look.

Reply:
Fixed